docs/django-apps.md

```diff
@@ -0,0 +1,67 @@
+# Using Caluma as django apps
+
+## Disclaimer
+
+It is highly recommended to use Caluma as a dedicated service. However, there are usecases, where
+it might make sense to integrate Caluma into another django project.
+
+If you just want to get Caluma up and running, please see the documentation about [setting up
+the Caluma service](configuration.md).
+
+## Installation
+
+Caluma is on PyPI, so you can just
+
+```shell
+pip install caluma
+```
+
+
+## Configuration
+
+### Installed apps
+
+Add the caluma apps you want to use to your `INSTALLED_APPS`:
+
+```python
+INSTALLED_APPS = [
+    # ...
+    # Caluma and it's dependencies:
+    "caluma.caluma_core.apps.DefaultConfig",
+    "caluma.caluma_user.apps.DefaultConfig",
+    "caluma.caluma_form.apps.DefaultConfig",
+    "caluma.caluma_workflow.apps.DefaultConfig",
+    "caluma.caluma_data_source.apps.DefaultConfig",
+    "graphene_django",
+    "localized_fields",
+    "psqlextra",
+    "simple_history",
+    # ...
+]
+```
+
+### Settings
+
+Import the caluma settings at the top of your `DJANGO_SETTINGS_MODULE`:
+
```

Comment:
Add a note that this only contains the settings that are required in addition to the regular django settings.

(Possibly also mention again to configure the database to use the `psqlextra` backend)